Guacamole/Assets/Scripts/GameControlScript.cs

```diff
@@ -5,7 +5,7 @@
 
 public class GameControlScript : MonoBehaviour
 {
-    public GameObject life1, life2, life3, gameover;
+    public GameObject life1, life2, life3;
```

Comment:
Why is gameover removed?

Reply:
I'm assuming it's moved somewhere else,  but I can't really find it.

Reply:
it's because gameover text is now on the gameover scene, not on the playscene.